scalikejdbc-streams/src/main/scala/scalikejdbc/streams/StreamReadySQL.scala

```diff
@@ -15,9 +15,9 @@ case class StreamReadySQL[A] private (
 
   private[streams] lazy val extractor: (WrappedResultSet) => A = underlying.extractor
 
-  private[streams] lazy val statement: String = underlying.statement
-  private[streams] lazy val rawParameters: Seq[Any] = underlying.rawParameters
-  private[streams] lazy val parameters: Seq[Any] = underlying.parameters
+  private[streams] def statement: String = underlying.statement
```

Comment:
extractor, fetchSize, tags, and queryTimeout are not `val`

- https://github.com/scalikejdbc/scalikejdbc/blob/3.0.1/scalikejdbc-core/src/main/scala/scalikejdbc/SQL.scala#L155
- https://github.com/scalikejdbc/scalikejdbc/blob/3.0.1/scalikejdbc-core/src/main/scala/scalikejdbc/SQL.scala#L215
- https://github.com/scalikejdbc/scalikejdbc/blob/3.0.1/scalikejdbc-core/src/main/scala/scalikejdbc/SQL.scala#L208
- https://github.com/scalikejdbc/scalikejdbc/blob/3.0.1/scalikejdbc-core/src/main/scala/scalikejdbc/SQL.scala#L238